config.yml

```diff
@@ -30,7 +26,7 @@ controller:
     seed: null                          # If null, random seed based on datetime is used
     dt: 0.02                              # sec
     mpc_horizon: 35                       # steps
-    CEM_NET_NAME: GRU-3IN-32H1-32H2-2OUT-0
+    predictor_specification: "ODE_TF"    # Can be "ODE", "ODE_TF", network/GP name (possibly with path) e.g. 'GRU-6IN-32H1-32H2-5OUT-0' or a name of a custom predictor. For more info see config_predictors in SI_Toolkit_ASF
```

Comment:
How would an example spec for the RNN predictor with custom network look?

Reply:
Not quite getting a question... You either specify the network name if it is "our" custom type. Then you can neural_default from config_predictors with model_name (and possibly path, if you give here a full path) overwritten. Or you need to reference here a completely different predictor by name (and it must be listed in config_predictors) and PredictorWrapper must take care on how to load such a predictor.